README.md

```diff
@@ -62,6 +62,21 @@ It's 100% Open Source and licensed under the [APACHE2](LICENSE).
 
 
 
+## Compliance
```

Comment:
The `build-harness` should not have a `## Compliance` section. See comment: https://github.com/cloudposse/build-harness/pull/256/files#r524569945

Reply:
done